aldryn_people/search_indexes.py

```diff
@@ -15,7 +16,8 @@ class PeopleIndex(get_index_base()):
     INDEX_TITLE = True
 
     def get_title(self, obj):
-        return obj.name
+        with switch_language(obj):
+            return obj.name
```

Comment:
Does this do anything? It would seem to me that if the obj is in language XX already, then returning its name would already be in lang XX, right?